source/_integrations/squeezebox.markdown

```
@@ -96,6 +96,51 @@ For example, to play an album from your collection, create an IFTTT applet like
 
 This can work with title search and basically any thing. The same wouldn't have worked by calling directly Squeezebox server as IFTTT cannot escape the text field.
 
+When specifying additional parameters in the Visual Editor, each parameter must be preceded by a hyphen and a space to correctly populate the array:
+
+For example, to create an automation to mute playback, use the command `mixer` and the parameter `muting`:
+
+| Row | Parameter | Description |
+| --- | --------  | ----------- |
+|  1  | - muting  | Toggle muting on / off 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)